readme.md

```diff
@@ -1,7 +1,7 @@
 # Hello world
 Replace `<your_account>` with your Github username and copy the links to Pull Request description:
-- [DEMO LINK](https://<your_account>.github.io/layout_hello-world/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_hello-world/report/html_report/)
+- [DEMO LINK](https://EhorSaprykin.github.io/layout_hello-world/)
+- [TEST REPORT LINK](https://EhorSaprykin.github.io/layout_hello-world/report/html_report/)
```

Comment:
don't forget to add this links on your PR description

Reply:
Done, ty